zulip_bots/README.md

```diff
@@ -107,9 +107,11 @@ Each bot library simply needs to do the following:
 
 - Define a class that supports the methods `usage`
 and `handle_message`.
+- Optionally add a class member `ACCEPT_EMPTY_MESSAGES`, to
+specify that empty messages will be passed to the bot.
```

Comment:
It would be awesome if you could also add this to `bots_guide.md`. The guide is quite outdated, but it will be the foundation for a major refactoring.